package.json

```diff
@@ -46,6 +46,7 @@
     "file-saver": "1.3.8",
     "font-awesome": "4.7.0",
     "history": "4.10.1",
+    "html-react-parser": "^3.0.7",
```

Comment:
Idea to add in a html parser as a way to render html elements in strings without having to use `dangerouslySetInnerHTML`. 

Would like feedback on the idea